string[] name
float64[] xMax
float64[] xMin
float64[] yMax
float64[] yMin
float64[] zMax
float64[] zMin